### PR TITLE
Updated test_amd64

### PR DESCRIPTION
**Pull request by anupriya-gupta**

### CODE REVIEW

**Comment by ltfish** [comment]
Wow, a lot of tests!

Can you please explain to me which new tests are replacing the old import tests that test if archinfo imports without capstone/keystone/unicorn/pyvex, and how?

**Comment by anupriya-gupta** [comment]
Hi Fish, I have not written the UT for imports yet. I'm just getting started with it and I will create a pull request as soon as I do that. 
I have raised this pull request just to get feedback if I have written it correctly or not.

**Comment by ltfish** [comment]
Gotcha. Maybe you shouldn't remote those test_import files then.

File: tests/test_amd64.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    import pyvex as _pyvex
+except ImportError:
+    _pyvex = None
+
+
+def test_arch_amd64():
+    endness = Endness.LE
+    assert ArchAMD64(endness)
+
+def test_arch_amd64_passes():
```
Comment (ltfish) [inline tests/test_amd64.py]: Two blank lines between global-level functions please.

File: tests/test_amd64.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+@raises(Exception)
+def test_keystone_x86_syntax_fails_4():
+    inst_1 = ArchAMD64(endness=Endness.LE)
+    inst_1.keystone_x86_syntax = 'ma'
+    nose.tools.assert_raises(Exception, inst_1.keystone_x86_syntax)
+
+
+def test_configure_keystone():
+    pass
```
Comment (ltfish) [inline tests/test_amd64.py]: You need a new line at the end of the file.

File: tests/test_amd64.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+def test_capstone_x86_syntax_passes():
+    inst_1 = ArchAMD64(endness=Endness.LE)
+
+    inst_1.capstone_x86_syntax = 'intel'
+    nose.tools.assert_raises(Exception, inst_1.capstone_x86_syntax)
+    inst_1.capstone_x86_syntax = 'at&t'
+    nose.tools.assert_raises(Exception, inst_1.capstone_x86_syntax)
+
+#Test raises one of expected exceptions to pass.
```
Comment (ltfish) [inline tests/test_amd64.py]: One space after `#`.

**Comment by anupriya-gupta** [comment]
Hey Fish, I have made the changes that you suggested. Please have a look.

**Comment by twizmwazin** [comment]
In order to get this merged, first can you un-delete the test_import files that you have deleted in this branch? Secondly, I see there are a few empty tests. You might want to double check if something was supposed to go there otherwise it would be best to remove them since they don't do anything. Thanks!

**Comment by twizmwazin** [comment]
Please fix the lint error, everything else looks good by me

**Comment by twizmwazin** [comment]
Merged manually :+1:

**Pull request closed without merging**